tests/test_data.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import torch
+from torchvision import transforms
+from torchvision.datasets import MNIST
+
+from tests import _PATH_DATA, _PROJECT_ROOT, _TEST_ROOT
+
+dataset_path = _PATH_DATA+"/processed"
+
+
+N_train = 60000
+N_test = 10000
+
+mist_transform = transforms.Compose([transforms.ToTensor()])
+
+@pytest.fixture
+def train_dataset():
```

Comment:
These need to reflect how we train our model in our project. The code in xgboost

Reply:
Fixed